src/blocks/keyboard_layout.rs

```diff
@@ -71,6 +80,8 @@
 //! ```
 
 use super::prelude::*;
+use regex::Regex;
+use std::collections::HashMap;
```

Comment:
`HashMap` is already in `super::prelude`

Reply:
I noticed that it appeared after I had to merge, and I saw it's used, so I didn't make the effort to double check why it was shown as a new line.. my bad, sorry. 

I've updated the PR, again..

Reply:
No problem